com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs

```diff
@@ -891,7 +891,7 @@ struct AmbientOcclusionAllocator
 
             public RTHandle Allocator(string id, int frameIndex, RTHandleSystem rtHandleSystem)
             {
-                return rtHandleSystem.Alloc(Vector2.one * scaleFactor, TextureXR.slices, filterMode: FilterMode.Point, colorFormat: GraphicsFormat.R32_UInt, dimension: TextureXR.dimension, useDynamicScale: true, enableRandomWrite: true, name: string.Format("{0}_AO Packed history_{1}", id, frameIndex));
+                return rtHandleSystem.Alloc(Vector2.one * scaleFactor, TextureXR.slices, filterMode: FilterMode.Point, colorFormat: GraphicsFormat.R32_SFloat, dimension: TextureXR.dimension, useDynamicScale: true, enableRandomWrite: true, name: string.Format("{0}_AO Packed history_{1}", id, frameIndex));
```

Comment:
Haven't we already discuss this exact opposite change in another PR in the past?

Reply:
You are right, my bad here.

The reason why it is confusing is that history is uint and main buffer is float.